recipes/skyfield/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "skyfield" %}
+{% set version = "0.9" %}
+{% set sha256 = "0f944793cfe63c897252c589ce29e4cc71dbafce1056d657517c02bfeb8e8a72" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - py2compat.patch
+
+build:
+  skip: true  # [win64]
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - sphinx 
+  run:
+    - python
+    - numpy
+    - astropy 
+    - pytz 
+    - lxml 
+    - html5lib 
+    - beautifulsoup4
+
+test:
+  requires:
+    - mock
+    - nose
+    - jplephem
+    - sgp4
+  imports:
+    - skyfield
+    - skyfield.data
+    - skyfield.tests
+  commands:
+    - nosetests -v skyfield.tests.test_functions
+    - nosetests -v skyfield.tests.test_angles
+
+about:
+  home: http://rhodesmill.org/skyfield/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Elegant astronomy for Python'
+
+  description: |
+    Skyfield is a pure-Python astronomy package that is compatible with both 
+    Python 2 and 3 and makes it easy to generate high precision research-grade 
+    positions for planets and Earth satellites.  
+  doc_url: http://rhodesmill.org/skyfield/toc.html
+  dev_url: https://github.com/skyfielders/python-skyfield
+
+extra:
+  recipe-maintainers:
+    - jochym
+    - brandon-rhodes
```

Comment:
He agreed at the end of this PR discussion https://github.com/skyfielders/python-skyfield/pull/77 . But I suppose it is better to wait for his confirmation here. He would be potentially helpful to make hist two packages (sgp4, skyfield) compatible with win builds.